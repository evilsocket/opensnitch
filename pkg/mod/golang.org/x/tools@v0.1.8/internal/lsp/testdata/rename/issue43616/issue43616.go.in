package issue43616

type foo int //@rename("foo","bar"),prepare("oo","foo","foo")

var x struct{ foo } //@rename("foo","baz")

var _ = x.foo //@rename("foo","quux")
