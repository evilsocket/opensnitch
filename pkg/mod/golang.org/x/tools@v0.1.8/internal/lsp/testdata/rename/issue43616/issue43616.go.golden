-- bar-rename --
package issue43616

type bar int //@rename("foo","bar"),prepare("oo","foo","foo")

var x struct{ bar } //@rename("foo","baz")

var _ = x.bar //@rename("foo","quux")

-- baz-rename --
can't rename embedded fields: rename the type directly or name the field
-- quux-rename --
can't rename embedded fields: rename the type directly or name the field
